docs/install_guides/installing-openvino-linux.md

```diff
@@ -42,91 +34,87 @@ Optimized for these processors:
    * For regular users: `/home/<USER>/intel/openvino_<version>/`
 
-   For simplicity, a symbolic link to the latest installation is also created: `/opt/intel/openvino_2021/` or `/home/<USER>/intel/openvino_2021/`
+   For simplicity, a symbolic link to the latest installation is also created: `/opt/intel/openvino_2022/` or `/home/<USER>/intel/openvino_2022/`
 
-8. **Optional**: You can choose **Customize** to change the installation directory or the components you want to install.
-   > **NOTE**: If there is an OpenVINO™ toolkit version previously installed on your system, the installer will use the same destination directory for the next installation. If you want to install a newer version to a different directory, you need to uninstall the previously installed versions.
+8. **Optional**: You can choose **Customize** to choose the components you want to install. You can choose between Inference Engine and Deep Learning Workbench, however you have to choose at least one of them.
    
-   > **NOTE**: The Intel® Media SDK component is always installed in the `/opt/intel/mediasdk` directory regardless of the OpenVINO installation path chosen.
+   > **NOTE**: From pull-down menu you can choose which Inference Engine version you want to install with regard to the hardware.  
 
-9. The **Finish** screen indicates that the core components have been installed:
-   ![](../img/openvino-install-linux-04.png)
+9. After consenting to the license agreement, you will be asked to consent to the collection oy your information. You don't have to consent to be able to continue the installation.
 
-   Once you click **Finish** to close the installation wizard, a new browser window will open with this documentation. It jumps to the section with your next installation steps.
+10. The following screen will present you with any libraries and packages that are required for successful OpenVINO installation. Click **Install** button and all missing dependencies will be installed on your machine. Once all installations are ready, click **Finish** button to go to **Installed Products** screen.
+
+11. The **Finish** screen indicates that the OpenVINO toolkit has been installed.
+   You can now close the installation wizard with exit button in the top right-hand corner of the screen or **Finish** button. However, you can also choose from the following options:
+    * **Get Started Guide** - will redirect you to our documentation site in a browser.
+    * **Modify** - will offer 4 more options:
+      *  **Add/Remove components** - will give you a possibility of installing and uninstalling components.
+      *  **Download a customized offline package** - allows you to download a package with components of your selection.
+      *  **Repair** - will reinstall the OpenVino toolkit after confirmation.
+      *  **Uninstall** - will let you uninstall the OpenVino toolkit after confirmation. This action cannot be reversed.
```

Comment:
I believe this should not be part of this document. Instead of there it would be better to suggest user to follow Getting Started Guide, Release notes or open install directory by link.